### PR TITLE
Update docs for authorize signer key methods in ReleaseGold

### PR DESCRIPTION
**Pull request by pranaymohan**
### Description

Updates docs on ReleaseGold.sol

### Tested

N/A

### Other changes

N/A

### Related issues

- Fixes #[issue number here]

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### CODE REVIEW

File: packages/protocol/contracts/governance/ReleaseGold.sol [hunk 1]
```diff
@@ -483,8 +483,8 @@ contract ReleaseGold is UsingRegistry, ReentrancyGuard, IReleaseGold, Initializa
    * @param v The recovery id of the incoming ECDSA signature.
    * @param r Output value r of the ECDSA signature.
    * @param s Output value s of the ECDSA signature.
-   * @dev The v,r and s signature should be a signed message by the beneficiary
-   *      encrypting the authorized address.
+   * @dev The v,r and s signature should be signed by the authorized signer
+   *      key, with the beneficiary address as the message.
```
Comment (asaj) [inline packages/protocol/contracts/governance/ReleaseGold.sol]: Actually the RG contract address should be the message, here and below

**Comment by barbaraliau** [comment]
@pranaymohan is this PR still relevant?

File: packages/protocol/contracts/governance/ReleaseGold.sol (hunk 1 above)
Comment (nategraf) [reply packages/protocol/contracts/governance/ReleaseGold.sol]:
```suggestion
   *      key, with the ReleaseGold contract address as the message.
```

File: packages/protocol/contracts/governance/ReleaseGold.sol [hunk 2]
```diff
@@ -544,8 +544,8 @@ contract ReleaseGold is UsingRegistry, ReentrancyGuard, IReleaseGold, Initializa
...
```
Comment (nategraf) [inline packages/protocol/contracts/governance/ReleaseGold.sol]:
```suggestion
   *      key, with the ReleaseGold contract address as the message.
```

File: packages/protocol/contracts/governance/ReleaseGold.sol [hunk 3]
```diff
@@ -567,8 +567,8 @@ contract ReleaseGold is UsingRegistry, ReentrancyGuard, IReleaseGold, Initializa
    * @param r Output value r of the ECDSA signature.
    * @param s Output value s of the ECDSA signature.
    * @param ecdsaPublicKey The ECDSA public key corresponding to `signer`.
-   * @dev The v,r and s signature should be a signed message by the beneficiary
-   *      encrypting the authorized address.
+   * @dev The v,r and s signature should be signed by the authorized signer
+   *      key, with the beneficiary address as the message.
```
Comment (nategraf) [inline packages/protocol/contracts/governance/ReleaseGold.sol]:
```suggestion
   *      key, with the ReleaseGold contract address as the message.
```

File: packages/protocol/contracts/governance/ReleaseGold.sol [hunk 4]
```diff
@@ -595,8 +595,8 @@ contract ReleaseGold is UsingRegistry, ReentrancyGuard, IReleaseGold, Initializa
    *   proof of possession. 96 bytes.
    * @param blsPop The BLS public key proof-of-possession, which consists of a signature on the
    *   account address. 48 bytes.
-   * @dev The v,r and s signature should be a signed message by the beneficiary
-   *      encrypting the authorized address.
+   * @dev The v,r and s signature should be signed by the authorized signer
+   *      key, with the beneficiary address as the message.
```
Comment (nategraf) [inline packages/protocol/contracts/governance/ReleaseGold.sol]:
```suggestion
   *      key, with the ReleaseGold contract address as the message.
```

File: packages/protocol/contracts/governance/ReleaseGold.sol [hunk 5]
```diff
@@ -628,8 +628,8 @@ contract ReleaseGold is UsingRegistry, ReentrancyGuard, IReleaseGold, Initializa
    * @param v The recovery id of the incoming ECDSA signature.
    * @param r Output value r of the ECDSA signature.
    * @param s Output value s of the ECDSA signature.
-   * @dev The v,r and s signature should be a signed message by the beneficiary
-   *      encrypting the authorized address.
+   * @dev The v,r and s signature should be signed by the authorized signer
+   *      key, with the beneficiary address as the message.
```
Comment (nategraf) [inline packages/protocol/contracts/governance/ReleaseGold.sol]:
```suggestion
   *      key, with the ReleaseGold contract address as the message.
```

**Review by nategraf — APPROVED** [review]

**Comment by pranaymohan** [comment]
sorry I missed your comment @barbaraliau !

**Pull request merged**